src/.htaccess

```diff
@@ -2,5 +2,6 @@ RewriteEngine On
 RewriteRule ^camel/(.*)$ http://camel.apache.org/$1 [R=301,L]
 RewriteRule ^nms.*$ https://activemq.apache.org/components/nms/ [R=301,L]
 RewriteRule ^cms.*$ https://activemq.apache.org/components/cms/ [R=301,L]
+RewriteRule ^artemis/docs/(.*)$ https://activemq.apache.org/components/artemis/documentation/$1 [R=301,L]
```

Comment:
not sure if this should be (.*) or just .* and if the $1 is needed or not.

Reply:
@michaelandrepearce I think that looks good. I'm new to mod_rewrite myself. :)